kotlintest-assertions/src/main/kotlin/io/kotlintest/throwablehandling/AnyThrowableHandling.kt

```diff
@@ -0,0 +1,49 @@
+package io.kotlintest.throwablehandling
+
+import io.kotlintest.Failures
+
+/**
+ * Verifies that a block of code throws any [Throwable]
+ *
+ * Use function to wrap a block of code that you want to verify that throws any kind of [Throwable], where using
+ * [shouldThrowAny] can't be used for any reason, such as an assignment of a variable.
+ *
+ * If you want to verify a specific [Throwable], use [shouldThrowExactlyUnit].
+ *
+ * If you want to verify a [Throwable] and any subclass, use [shouldThrowUnit].
+ *
+ * @see [shouldThrowAny]
+ */
+inline fun shouldThrowAnyUnit(block: () -> Unit) = shouldThrowAny(block)
+
+/**
+ * Verifies that a block of code throws any [Throwable]
+ *
+ * Use function to wrap a block of code that you want to verify that throws any kind of [Throwable].
+ *
+ * If you want to verify a specific [Throwable], use [shouldThrowExactly].
+ *
+ * If you want to verify a [Throwable] and any subclasses, use [shouldThrow]
+ *
+ *
+ * **Attention to assignment operations:**
+ *
+ * When doing an assignment to a variable, the code won't compile, because an assignment is not of type [Any], as required
+ * by [block]. If you need to test that an assignment throws a [Throwable], use [shouldThrowAnyUnit] or it's variations.
```

Comment:
Nice. I'd slightly reword to say that an assigment is not an expression but a statement, and therefore has no return value.

Reply:
Good one.

Reply:
done.